api/src/main/java/org/openmrs/module/fhir/api/impl/MedicationRequestServiceImpl.java

```diff
@@ -136,25 +139,15 @@ public MedicationRequest updateFHIRMedicationRequest(MedicationRequest medicatio
         FHIRMedicationRequestUtil.copyObsAttributes(incomingDrugOrder, generatedDrugOrder, errors);
         if (generatedDrugOrder != null) { //medication request update
             if (!errors.isEmpty()) {
-                StringBuilder errorMessage = new StringBuilder("The request cannot be processed due to the " +
+                String errorMessage = ErrorUtil.generateErrorMessage(errors, "The request cannot be processed due to the " +
```

Comment:
You can use FHIRConstants.REQUEST_ISSUE_LIST here.

Reply:
Nica catch, thanks